src/main/java/mate/academy/service/UserService.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.service;
+
+import java.util.Optional;
+import mate.academy.model.User;
+
+public interface UserService {
+    User add(User user);
+
+    Optional<User> findByEmail(String email);
```

Comment:
```suggestion
    User getByEmail(String email);
```
Let's better to return on Service layer.

Reply:
we have this in our tasks 
![image](https://github.com/mate-academy/hibernate-user-service-hw/assets/73792778/cfc00d0b-fa96-4bda-a7a5-6256e48f6b8f)

Reply:
@SkillfulDev there is no silver bullet, but https://stackoverflow.com/a/43830087